morecantile/models.py

```diff
@@ -1021,61 +1025,58 @@ def tile(
         x, y = self.xy(lng, lat, truncate=truncate)
         return self._tile(x, y, zoom, ignore_coalescence=ignore_coalescence)
 
-    def _ul(self, *tile: Tile) -> Coords:
+    def _ul(self, tile: Tile) -> Coords:
```

Comment:
I wonder if it would be just as fast to create a tile type here that can either be a Tile or tuple[int,int,int] here and elsewhere so that the api can be kept more similar to what it was before, basically users would keep their tile coordinates packed in a tuple rather than expanding them.